ipn/ipnlocal/local.go

```diff
@@ -2216,7 +2216,7 @@ func (b *LocalBackend) peerAPIServicesLocked() (ret []tailcfg.Service) {
 // possibly after mangling the given hostinfo.
 //
 // TODO(danderson): we shouldn't be mangling hostinfo here after
-// painstakingly constructing it in twelvety other places.
+// painstakingly constructing it in twelve other places.
```

Comment:
I can't tell if this is being cute, in which case I suppose I could leave it alone. If it isn't a precise number, might I suggest "many"?

Reply:
It's being cute